tests/test_well/test_well.py

```diff
@@ -353,6 +353,48 @@ def test_mask_shoulderbeds_get_bseries(logseries, nsamples, expected):
         assert results.equals(expected)
 
 
+@pytest.mark.parametrize(
+    "dep, lseries, distance, expected",
+    [
+        (
+            [1.0],
+            [1],
+            1.2,
+            [False],
+        ),
+        (
+            [1.0, 2.0, 3.0, 4.0, 5.0, 6.0],
+            [1, 1, 1, 2, 2, 2],
+            0.7,
+            [False, False, True, True, False, False],
+        ),
+        (
+            [1.0, 2.0, 3.0, 4.0, 5.0, 6.0],
+            [1, 1, 1, 2, 3, 2],
+            1.2,
```

Comment:
There should be a test with `distance == 1.0`

Reply:
(assuming that is a tricky number, but looking at the expected booleans it might not be true that 1.0 is a critcal value?)

Reply:
Good point. 0.5 will be a tricky number since the transition here is in the middle, e.g. at 3.5. I added a test and modified the code so that distance is >=, not > which is more intuitive. Hence 0.5 will trigger a mask while 0.49999 will not.